include/z64actor.h

```diff
@@ -38,11 +39,50 @@ typedef struct {
     /* 0x1C */ ActorFunc draw; // Draw function
 } ActorInit; // size = 0x20
 
-typedef enum {
-    /* 0 */ ALLOCTYPE_NORMAL,
-    /* 1 */ ALLOCTYPE_ABSOLUTE,
-    /* 2 */ ALLOCTYPE_PERMANENT
-} AllocType;
+/**
+ * @see ACTOROVL_ALLOC_ABSOLUTE
+ */
+#define ACTOR_OVERLAY_ABSOLUTE_SPACE_SIZE 0x27A0
```

Comment:
```suggestion
#define ACTOROVL_ABSOLUTE_SPACE_SIZE 0x27A0
```
Should it be this for consistency?

Reply:
I thought about it and I don't know either if it would be better. The other names use actorovl_ more to keep it short than to be different, but they are also different so I don't think consistency matters.

All in all, I don't know

Anyone wanna take a decision? :pray:

Reply:
kinda torn as well. I think it makes sense for it to share the same prefix as the type, they at least concern the same specific parts of the zelda heap in relation to actor overlays